### DISCUSSION TITLE
Lorien doesn't scale properly in Ubuntu

### DISCUSSION THREAD

**Discussion by labiol** [mbrlabs/Lorien#83] [Ideas]
Ubuntu 20.04, display -> scale settings = 125%. 
Lorien doesn't scale and menu, icon are very tiny. 
Btw. fantastic software.

**Comment by mbrlabs** [comment]
@labiol afaik these scaling settings only apply to GTK/Qt programs, not to custom UI rendered on the GPU using OpenGL.

But Lorien now supports UI scaling for high-dpi monitors (see: https://github.com/mbrlabs/Lorien/pull/102). It's not officially released yet but you can donwload a snapshot from here: https://github.com/mbrlabs/Lorien/actions/runs/1688060471 (Linux binaries at the bottom of the page). To enable it you have to open the settings, go into `Appearance` and set the UI scaling to custom. Then you can change the scale to 1.25.
